### DISCUSSION TITLE
Preact with remotion.

### DISCUSSION THREAD

**Discussion by HenriqueMatias** [remotion-dev/remotion#1811] [General]
Considering how similar react and preact are, I was wondering if it is possible to use them together.
Or if there is any intention to create a remotion-preact.
Thank you in advance :)

**Comment by JonnyBurger** [comment]
No intention to implement it currently, since React is a peer dependency of Remotion, it would make sense if you also used React for your code so the version can be shared.

I assume one of the main motivations for Preact would be to have lower bundle size, but I think we cannot make the core agnostic, so you would end up with both React and Preact unfortunately.

**Reply by HenriqueMatias** [reply]
just for curiosity. What features of react are so important for remotion core?

**Reply by JonnyBurger** [reply]
@HenriqueMatias Probably none, I could see it also be done with Preact. But we already went with React and making the core agnostic is challenging.
